zubhub_backend/Makefile

```diff
@@ -107,7 +107,17 @@ admin-user:
 	docker-compose -f docker-compose.yml exec web bash -c "rm /dummy.sh"
 .PHONY: admin-user
 
-init: .env start migrate admin-user  ## Initialize docker-compose containers
+add-theme:
+	@echo "==="
+	@echo "== Add a theme to the database =="
+	@echo "==="
+	docker-compose -f docker-compose.yml exec web bash -c "echo 'echo \"from zubhub.models import Theme; \
+	Theme.objects.create(Theme_Name=\\\"Theme Zubhub\\\", status=\\\"1\\\") if not Theme.objects.filter(Theme_Name=\\\"Theme Zubhub\\\").exists() else None\" | python /zubhub_backend/zubhub/manage.py shell' > /theme.sh"
```

Comment:
Can we rename the theme to Default instead of `Theme Zubhub`?

Reply:
alright @tuxology